app/workers/my_commodities_subscription_worker.rb

```diff
@@ -0,0 +1,28 @@
+class MyCommoditiesSubscriptionWorker
+  include Sidekiq::Worker
+
+  def perform(date = Time.zone.yesterday)
```

Comment:
Should this be something that can be passed via redis? Like an iso date and not a complex object?